Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pydantic'...
Compiling './lib/python3.5/site-packages/pydantic/exceptions.py'...
***   File "./lib/python3.5/site-packages/pydantic/exceptions.py", line 52
    v = ' '.join(f'{k}={e.get(k)}' for k in E_KEYS if e.get(k))
                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pydantic/fields.py'...
***   File "./lib/python3.5/site-packages/pydantic/fields.py", line 38
    self.name: str = name
             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pydantic/main.py'...
***   File "./lib/python3.5/site-packages/pydantic/main.py", line 94
    raise ValueError(f'"{self.__class__.__name__}" object has no field "{name}"')
                                                                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pydantic/types.py'...
***   File "./lib/python3.5/site-packages/pydantic/types.py", line 31
    min_length: int = None
              ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pydantic/utils.py'...
***   File "./lib/python3.5/site-packages/pydantic/utils.py", line 86
    raise ImportError(f'"{dotted_path}" doesn\'t look like a module path') from e
                                                                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pydantic/validators.py'...
***   File "./lib/python3.5/site-packages/pydantic/validators.py", line 54
    raise ValueError(f'size not in range {model.config.min_number_size} to {model.config.max_number_size}')
                                                                                                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pydantic-0.1.egg-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
